### ISSUE TITLE
Can I access the cache across different script?

### ISSUE THREAD

**Issue by tungrix** [node-cache/node-cache#312]
Let's say I am declaring an `obj` in test1.js.
```javascript
obj = { my: "Special", variable: 42 };
 
success = myCache.set( "myKey", obj, 10000 );
```
Is it possible to access `"myKey"` in test2.js during running? 

If yes, how can I do it? 

If not, how to access `"myKey"` in some child nested functions? Through multiple passing as the parameter?

**Comment by tungrix** [comment]
Thank you so much.

If I want to apply something like refreshing the token every 9 hours, is it possible for me to do it like this inside cache.js, or is there any better way to do it?
```javascript
const cache = new NodeCache();
cache.set("token", await getToken(), 9*60*60);

cache.on("del", async function (key, value) {
  cache.set("token", await getToken(), 9*60*60);
  });
```

**Comment by AggelosLua** [comment]
Yes, that is possible. Inside that file on in the file that you require it in, you can still bind events. Although, to answer your question, it is generally better to either place them in the file where you mainly use nodecache in, or if you would like, for better code organization, make a function like

```js
const cache = new NodeCache();
// . . . 
function registerEvents() {
	cache.cache.on("del", (key, value) => {
             cache.cache.set(key, "...");
    });
}
```

And to answer the second question, `stdTTL` in the constructor option, applies to every `cache.set()` **BUT**m if you have the stdTLL in the constructor aswell as in a `cache.set()`, its going to use that one but if you dont use the `stdTTL` constructor option and use it in the `cache.set()`, it will utilize that one.
